portals/publisher/source/src/app/data/AuthManager.js

```diff
@@ -164,6 +164,25 @@ class AuthManager {
         }
     }
 
+    /**
+     *
+     * Check whether user is a readOnly user
+     * @static
+     * @returns {Boolean} isReadOnlyUser
+     * @memberof AuthManager
+     */
+    static isReadOnlyUser() {
+        if (AuthManager.getUser() === null) {
+            return false;
+        } else {
+            const arrayLength = AuthManager.getUser().scopes.length;
+            if (arrayLength === 2) {
+                return AuthManager.getUser().scopes.includes('apim:api_view', 'openid');
```

Comment:
`Array.includes` does not accept multiple values, We have to use either lodash intercept or a simplified version of it using pure JS

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes